src/main/java/com/thebois/models/beings/Pawn.java

```diff
@@ -22,19 +28,24 @@
      * @param destination   initial destination to travel to.
      * @param random        the generator of random numbers.
      * @param pathFinder    The generator of paths to positions in the world.
+     * @param finder          Finder used to locate things in the world
```

Comment:
style

No point repeating the name/type of the parameter in the javadoc. Just "Used to locate things in the world." is enough to describe it.

Also, missing period.

Reply:
And missing capitalization on two of the other parameter javadocs.